src/main/java/seedu/recruittrackpro/model/person/Comment.java

```diff
@@ -8,7 +8,7 @@
  * Guarantees: immutable; is valid as declared in {@link #isValidComment(String)}
  */
 public class Comment {
-    public static final String MESSAGE_CONSTRAINTS = "Comments should not exceed 500 characters";
+    public static final String MESSAGE_CONSTRAINTS = "Comments should not be empty and must not exceed 500 characters";
 
     public static final String VALIDATION_REGEX = ".{0,500}";
```

Comment:
just curious if this is intentional change? this implies that all candidates in RecruitTrackPro needs to have a comment? Do we consider cases where there's no comments (like tags) when adding a new candidate? If this is an intentional change, do note that the `VALIDATION_REGEX` should be 
```suggestion
    public static final String MESSAGE_CONSTRAINTS = "Comments should not be empty and must not exceed 500 characters";

    public static final String VALIDATION_REGEX = ".{1,500}";

Reply:
This was an intentional change. I was trying to give a clearer error message. Because when the command `find c/` is entered without any comment keywords, the previous error message was "Comments must not exceed 500 characters" which did not make sense when no keywords was inputed.